shared/challenge.go

```diff
@@ -4,10 +4,19 @@ import "github.com/spacemeshos/sha256-simd"
 
 type Challenge []byte
 
-var ZeroChallenge = make(Challenge, 0)
+var (
+	ZeroChallenge = make(Challenge, 0)
+	message []byte
```

Comment:
is it a message? maybe a better name that will indicate why we added this. or a name and a comment

Reply:
I'd rather call it `buffer`, to explain its functionality.